backend/src/main/resources/graphql/admin.graphqls

```diff
@@ -10,15 +10,16 @@ extend type Mutation {
   createDeviceType(input: CreateDeviceType!): DeviceType
   updateDeviceType(input: UpdateDeviceType!): DeviceType
   createSpecimenType(input: CreateSpecimenType!): SpecimenType
-  addUser( # not actually used in the admin console (currently)
+  addUser( # used in AddOrganizationAdminFormContainer
     name: NameInput
-    firstName: String
-    middleName: String
-    lastName: String
-    suffix: String
-    email: String!
-    organizationExternalId: String!
-    role: Role!
+    firstName: String @deprecated(reason: "use user parameter instead")
+    middleName: String @deprecated(reason: "use user parameter instead")
+    lastName: String @deprecated(reason: "use user parameter instead")
+    suffix: String  @deprecated(reason: "use user parameter instead")
+    email: String  @deprecated(reason: "use user parameter instead")
```

Comment:
should we keep `email`, `organizationExternalId`, and `role` as required though while we maintain backward compatibility

Reply:
no, if we want to keep the second PR as frontend only. So [the PR](https://github.com/CDCgov/prime-simplereport/pull/5733/files) that is coming after this will start passing the userObject and will drop all these other granular fields, so I need them to not be mandatory anymore.

Reply:
Ahhh got it! Makes sense -- thank you for explaining 😸